project.clj

```diff
@@ -22,7 +22,7 @@
                  [prismatic/schema]
                  [slingshot]
 
-                 [puppetlabs/jruby-deps "9.4.7.0-1"]
+                 [puppetlabs/jruby-deps "9.4.8.0-2-SNAPSHOT"]
```

Comment:
we want the tegged version https://github.com/puppetlabs/jruby-deps/releases/tag/9.4.8.0-1